<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>journaliste_mission</class>
 <widget class="QDialog" name="journaliste_mission">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>841</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="gestion_journaliste">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 22pt &quot;Roboto&quot;;
border-image: url(:/xxx.png);
background-color: transparent;
background: none;
border: none;
background-repeat: none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="gestion_des_mission">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>150</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 22pt &quot;Roboto&quot;;

border-image: url(:/gesmiss.png);
background-color: transparent;
background: none;
border: none;
background-repeat: none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="animation">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-style:italic;&quot;&gt;&lt;br/&gt;gestion des journalistes &lt;br&gt; et missions&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../cccc-01.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>831</width>
       <height>401</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Roboto&quot;;
border-image: url(:/click.mp3);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <widget class="QPushButton" name="pb_ajouter_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>280</y>
         <width>181</width>
         <height>34</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/add.png);


background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_8">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>651</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;Roboto&quot;;</string>
            </property>
            <property name="text">
             <string>id journaliste:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>nom :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>prenom :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>fonction :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>diplome :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLineEdit" name="id"/>
          </item>
          <item>
           <widget class="QLineEdit" name="le_nom_2"/>
          </item>
          <item>
           <widget class="QLineEdit" name="le_prenom_2"/>
          </item>
          <item>
           <widget class="QLineEdit" name="le_fonction_2"/>
          </item>
          <item>
           <widget class="QLineEdit" name="le_diplome_2"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="return_3">
       <property name="geometry">
        <rect>
         <x>704</x>
         <y>330</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/return.png);
background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Gestion des journalistes</string>
      </attribute>
      <widget class="QWidget" name="horizontalLayoutWidget_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>221</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Roboto&quot;;</string>
          </property>
          <property name="text">
           <string>recherche</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="id_rechercher_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>401</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QTableView" name="tab_journaliste_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_12">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>50</y>
         <width>341</width>
         <height>221</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_27">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;Roboto&quot;;</string>
            </property>
            <property name="text">
             <string>id journaliste:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_28">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;Roboto&quot;;</string>
            </property>
            <property name="text">
             <string>nom :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_29">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;Roboto&quot;;</string>
            </property>
            <property name="text">
             <string>prenom:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_30">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;Roboto&quot;;</string>
            </property>
            <property name="text">
             <string>fonction:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_31">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;Roboto&quot;;</string>
            </property>
            <property name="text">
             <string>diplome :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLineEdit" name="le_id_6"/>
          </item>
          <item>
           <widget class="QLineEdit" name="le_nom_5"/>
          </item>
          <item>
           <widget class="QLineEdit" name="le_prenom_5"/>
          </item>
          <item>
           <widget class="QLineEdit" name="le_fonction_4"/>
          </item>
          <item>
           <widget class="QLineEdit" name="le_diplome_4"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="recherche_tab_2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>270</y>
         <width>411</width>
         <height>79</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="recherchep_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>301</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Roboto&quot;;

border-image: url(:/recherche.png);

background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="trieprenom">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: transparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/prenom.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="tri_p_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>21</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: transparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/nom.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pdf_p_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/pdf.png);
background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="afficherp_2">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>20</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Roboto&quot;;
border-image: url(:/afficher.png);

background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="modifierp_2">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>20</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Roboto&quot;;
background-color: transparent;
border-image: url(:/modifier.png);
background: none;
border: none;
background-repeat: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_2">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>20</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Roboto&quot;;
border-image: url(:/delete.png);


background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Mailling</string>
      </attribute>
      <widget class="QFrame" name="statv">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>561</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>821</width>
       <height>421</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Roboto&quot;;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <widget class="QPushButton" name="pb_ajouter_3">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>260</y>
         <width>181</width>
         <height>34</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: transparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/add.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_15">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>651</width>
         <height>151</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;Roboto&quot;;</string>
            </property>
            <property name="text">
             <string>id mission</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>libelle mission</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>description mission</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_34">
            <property name="text">
             <string>confidentialite</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QLineEdit" name="id_mission"/>
          </item>
          <item>
           <widget class="QLineEdit" name="libelle"/>
          </item>
          <item>
           <widget class="QLineEdit" name="description"/>
          </item>
          <item>
           <widget class="QLineEdit" name="confidentialite"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="return_4">
       <property name="geometry">
        <rect>
         <x>704</x>
         <y>330</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: transparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/return.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Gestion des missions</string>
      </attribute>
      <widget class="QWidget" name="horizontalLayoutWidget_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>211</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_35">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Roboto&quot;;</string>
          </property>
          <property name="text">
           <string>recherche</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="id_rechercher_3"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>391</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QTableView" name="tab_journaliste_3"/>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="recherche_tab_3">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>270</y>
         <width>391</width>
         <height>79</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_19">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>90</y>
         <width>351</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <item>
           <widget class="QLabel" name="label_37">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;Roboto&quot;;</string>
            </property>
            <property name="text">
             <string>id mission</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_38">
            <property name="text">
             <string>libelle mission</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_39">
            <property name="text">
             <string>description mission</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_40">
            <property name="text">
             <string>confidentialite</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <widget class="QLineEdit" name="id_miss_mod"/>
          </item>
          <item>
           <widget class="QLineEdit" name="libelle_miss_mod"/>
          </item>
          <item>
           <widget class="QLineEdit" name="description_mod"/>
          </item>
          <item>
           <widget class="QLineEdit" name="confidentialite_mod"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pdf_p_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: transparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/pdf.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="tri_p_3">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>20</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Roboto&quot;;

background-color: transparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/TRIE.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_36">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-70</y>
         <width>161</width>
         <height>209</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Roboto&quot;;</string>
       </property>
       <property name="text">
        <string>Affichage des mission</string>
       </property>
      </widget>
      <widget class="QPushButton" name="afficherp_3">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>20</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/afficher.png);

background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="modifierp_3">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>20</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/modifier.png);
background-color: transparent;
background: none;
border: none;
background-repeat: none;
font: 10pt &quot;Roboto&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_3">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>20</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: transparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/delete.png);
font: 10pt &quot;Roboto&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="recherchep_3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>300</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/recherche.png);
background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
